cloudtracker/__init__.py

```diff
@@ -228,12 +229,15 @@ def get_user_allowed_actions(aws_api_list, user_iam, account_iam):
 
     # Get permissions from groups
     for group in groups:
-        group_iam = pyjq.one('.GroupDetailList[] | select(.GroupName == "{}")'.format(group), account_iam)
+        group_iam = jmespath.search('GroupDetailList[] | [?GroupName == `{}`] | [0]'.format(group), account_iam)
+        if group_iam is None:
+            continue
         # Get privileges from managed policies attached to the group
         for managed_policy in group_iam['AttachedManagedPolicies']:
-            policy_filter = '.Policies[] | select(.Arn == "{}") | ' \
-                            '.PolicyVersionList[] | select(.IsDefaultVersion == true) | .Document'
-            policy = pyjq.one(policy_filter.format(managed_policy['PolicyArn']), account_iam)
+            policy_filter = 'Policies[?Arn == `{}`].PolicyVersionList[?IsDefaultVersion == true] | [0][0].Document'
+            policy = jmespath.search(policy_filter.format(managed_policy['PolicyArn']), account_iam)
+            if police is None:
```

Comment:
Looks like a typo here where `police` should be `policy`

Reply:
definitely a typo, sorry about that 🤣